File: problems/1293-three-consecutive-odds/three-consecutive-odds.js
/**
 * @param {number[]} arr
 * @return {boolean}
 */
const threeConsecutiveOdds = arr => {
    let count = 0;
    for (let i = 0; i < arr.length - 2; i++) {
        if (isOdd(arr[i]) && isOdd(arr[i + 1]) && isOdd(arr[i + 2])) return true;
    }
    return false;
};

const isOdd = num => {
    return num % 2 !== 0;
}

File: problems/1293-three-consecutive-odds/three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
      for i in range(len(arr) - 2):
        if self.isOdd(arr[i]) and self.isOdd(arr[i + 1])  and self.isOdd(arr[i + 2]):
            return True
      return False
        

    def isOdd(self, num:[int]) -> bool:
        return num % 2 != 0